packages/api/lib/lzards.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const { getRequiredEnvVar } = require('@cumulus/common/env');
+const { getSecretString } = require('@cumulus/aws-client/SecretsManager');
+const { getLaunchpadToken } = require('@cumulus/launchpad-auth');
+const got = require('got');
+const Logger = require('@cumulus/logger');
+
+const log = new Logger({ sender: 'api/lib/lzards' });
+
```

Comment:
`lzards` request can use different launchpad credentials `lzards_launchpad_certificate` and `lzards_launchpad_passphrase`, although we use the same one for CMR ingest in our example.
https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/main.tf#L133-L134

Reply:
If we have both`lzards_launchpad_passphrase` and `launchpad_passphrase` configured for `cumulus` module, it's confusing which launchpad_passphrase is actually used here.

Reply:
i'm not sure how best to address that confusion. For now, I renamed the lambda environment variables to be prefixed with `lzards_` like `lzards_launchpad_passphrase_secret_name`. I updated `lambdas.tf` to use `lzards_xxx` values rather than the non-`lzards_xxx` variables.